shell/renderer/electron_render_frame_observer.cc

```diff
@@ -58,10 +58,10 @@ ElectronRenderFrameObserver::ElectronRenderFrameObserver(
   // Initialise resource for directory listing.
   net::NetModule::SetResourceProvider(NetResourceProvider);
 
-  // App regions are only supported in the main frame.
-  auto* main_frame = frame->GetMainRenderFrame();
-  if (main_frame && main_frame == frame)
-    render_frame_->GetWebView()->SetSupportsAppRegion(true);
+  // In Chrome, app regions are only supported in the main frame.
+  // However, we need to support draggable regions on other
+  // local frames/windows, so extend support beyond the main frame.
+  render_frame_->GetWebView()->SetSupportsAppRegion(true);
```

Comment:
Reason for excluding child frames was devtools inspect gets broken, refs https://electronhq.slack.com/archives/CB6CG54DB/p1705578798763399?thread_ts=1705578659.730389&cid=CB6CG54DB. Please check if inspect continues to work with this change.

/cc @marekharanczyk